README.md

```diff
@@ -1,15 +1,111 @@
-# OpenCensus Core Node.js
+# OpenCensus Libraries for Node.js
 [![Gitter chat][gitter-image]][gitter-url]
 
-OpenCensus for Node.js is an implementation of OpenCensus, a toolkit for collecting application performance and behavior monitoring data. 
+OpenCensus Node.js is an implementation of OpenCensus, a toolkit for collecting application performance and behavior monitoring data. Right now OpenCensus for Node.js supports custom tracing and automatic tracing for HTTP and HTTPS.
 
 The library is in alpha stage and the API is subject to change.
 
 Please join [gitter](https://gitter.im/census-instrumentation/Lobby) for help or feedback on this project.
 
+## Installation
+
+Install OpenCensus with:
+
+```bash
+npm install @opencensus/nodejs
+```
+
+## Traces
```

Comment:
I think it would be better if info wasn't duplicated across this and the `@o/nodejs` package README as well. I would suggest cutting technical details from this doc, and linking to the other one instead.

Reply:
Ok, I removed the technical part and added a link to nodejs package. I also left the info about plugin, propagation and exporters to be like a "current state" of the library kinda thing. Let me know if there is anything that would be great to add.